### PR TITLE
Texture fix, Image Optimisation, Datapack layout update

### PR DESCRIPTION
**Pull request by PeterHurt**

### CODE REVIEW

File: datapacks/fleet/entities/battleship.json [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+      "regen": 10
+    },
     "movement": {
         "range": 1
       },
     "combat": {
-        "damage": "100",
-        "range": 4
+      "range": 4,
+      "attack": "100",
```
Comment (SamGarlick) [inline datapacks/fleet/entities/battleship.json:12]: This should be a number not a string. The parsing should have detected this - I will need to look into why it didn't

File: datapacks/fleet/entities/cruiser.json [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+      "regen": 8
+    },
     "movement": {
         "range": 2
       },
     "combat": {
-        "damage": "60",
-        "range": 4
+      "range": 4,
+      "attack": "60",
```
Comment (SamGarlick) [inline datapacks/fleet/entities/cruiser.json:12]: Same here

File: datapacks/fleet/entities/battleship.json (hunk 1 above)
Comment (PeterHurt) [reply datapacks/fleet/entities/battleship.json]: Thanks, fixed in 3dbbd96

File: datapacks/fleet/entities/cruiser.json (hunk 2 above)
Comment (PeterHurt) [reply datapacks/fleet/entities/cruiser.json]: As above, fixed in 3dbbd96

**Pull request merged**